LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Joren De Groof
```

Comment:
😅

Reply:
Do you want to keep the copyright? 🙂

Reply:
No no, this is ok. Funny we didn't change it immediately, same goes for all the url's in the readme to the CoC etc.